TheaterSchedule.DAL/Repositories/GoogleFormUrlRepository.cs

```diff
@@ -33,5 +37,11 @@ public UrlModel GetUrl()
                 UrlId = url.UrlId
             };
         }
+
+        public void Delete()
+        {
+            var existingUrl = db.FormUrl.First();

```

Comment:
i think that it will be better to update existing record instead of deleting and adding new